docs/maven-plugin.md

```diff
@@ -106,7 +106,7 @@ Parameters:
 ### static-code-analysis:findbugs
 
 Description:
-    Executes the `findbugs-maven-plugin` goal `findbugs` with a  ruleset file and configuration properties
+    Executes the `spotbugs-maven-plugin` goal `findbugs` with a  ruleset file and configuration properties

```

Comment:
Yes, it sounds strange, but is the same [goal ](https://github.com/hazendaz/findbugs-maven-plugin/blob/master/src/main/groovy/org/codehaus/mojo/findbugs/FindBugsMojo.groovy#L58).